python/requirements.txt

```diff
@@ -12,8 +12,8 @@ cloudpickle
 filelock
 frozenlist
 gpustat == 1.0.0b1
-grpcio >= 1.32.0, <= 1.43.0; python_version < '3.10'
-grpcio >= 1.42.0, <= 1.43.0; python_version >= '3.10'
+grpcio >= 1.32.0, <= 1.48.1; python_version < '3.10'
```

Comment:
Why can't we go above 1.48.1?

Reply:
This is the highest version currently published. Do you think I should remove the upper bound altogether?

Reply:
Yes, we should remove it so future higher versions can be supported automatically.